pipelines/bigcommerce.user.autologin.v1.json

```diff
@@ -4,19 +4,63 @@
     "id": "bigcommerce.user.autologin.v1",
     "public": true,
     "input": [
-      {"key": "sgxsMeta", "id": "1000", "optional": true},
-      {"key": "token", "id": "1001"}
+      {
```

Comment:
Why not using oneliners for that, as we do in all?! other repos?

Reply:
automatically adjusted as I comitted :(

Reply:
or maybe I am delusional, too many hours spent

Reply:
Waiting for the fixing commit. :)

Reply:
weird, I pushed it , but it's not here :D  I guess it is the automatic unraveling, it adjusts the code as the commit is made. Anywho disabled auto reformat :)